packages/datadog_session_replay/android/src/main/kotlin/com/datadoghq/flutter/sessionreplay/feature/FlutterSessionReplayFeature.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2025-Present Datadog, Inc.
+ */
+
+package com.datadoghq.flutter.sessionreplay.feature
+
+import android.content.Context
+import com.datadog.android.api.feature.Feature
+import com.datadog.android.api.feature.FeatureContextUpdateReceiver
+import com.datadog.android.api.feature.FeatureEventReceiver
+import com.datadog.android.api.feature.FeatureSdkCore
+import com.datadog.android.api.feature.StorageBackedFeature
+import com.datadog.android.api.net.RequestFactory
+import com.datadog.android.api.storage.EventType
+import com.datadog.android.api.storage.FeatureStorageConfiguration
+import com.datadog.android.api.storage.RawBatchEvent
+
+class FlutterSessionReplayFeature(
+    private val sdkCore: FeatureSdkCore,
+    private val onContextChanged: (Map<String, Any?>) -> Unit,
+    private val customEndpointUrl: String?
+) : StorageBackedFeature, FeatureEventReceiver, FeatureContextUpdateReceiver {
+
+    override val name = Feature.SESSION_REPLAY_FEATURE_NAME
+    override val storageConfiguration = STORAGE_CONFIGURATION
+
+    override val requestFactory: RequestFactory by lazy {
+        SegmentRequestFactory(
+            customEndpointUrl,
+            BatchesToSegmentsMapper(sdkCore.internalLogger)
+        )
+    }
+
+    override fun onInitialize(appContext: Context) {
+        sdkCore.setContextUpdateReceiver(
+            SESSION_REPLAY_FEATURE_NAME,
+            this
+        )
+    }
+
+    override fun onStop() {
+    }
+
+    override fun onReceive(event: Any) {
+    }
+
```

Comment:
```suggestion
```

Reply:
`onStop` is a required part of `StorageBackedFeature`. `onRecieve` can go away through as I don't need to inherit from `FeatureEventReceiver`